### DISCUSSION TITLE
feat: Support Stability Stable Audio Open

### DISCUSSION THREAD

**Discussion by jeremiahsb** [open-webui/pipelines#202] [Ideas]
**Describe the solution you'd like**
It would be great if open-webui was able to incorporate the newly released Stability AI Stable Audio capabilities to complement the existing multimedia capabilities.

https://huggingface.co/stabilityai/stable-audio-open-1.0
https://github.com/Stability-AI/stable-audio-tools

**Describe alternatives you've considered**
It's possible to run Stable Audio directly via the example script in the repo, but not as ideal.

**Comment by bannert1337** [comment]
It could be implemented as a tool in [Pipelines](https://github.com/open-webui/pipelines)
